sys/dev/ath/if_ath_usb_def.h

```diff
@@ -525,6 +525,7 @@ struct ath_usb_softc {
 	void				(*sc_node_free)(struct ieee80211com *,
 					    struct ieee80211_node *);
 	int				sc_key_tasks;
+	bool				interrupt;
```

Comment:
Not sure if `interrupt` name for this property is correct. Maybe `immediate_write`, `no_buffer_write` etc. would be better?

Reply:
I changed to no_buffer_write